src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +81,58 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={e => handleQueryChange(e.target.value)}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isQueryAbsent && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
```

Comment:
actual string is more readable
```suggestion
              {'Can\'t find a movie with such a title'}
              // or
              {`Can't find a movie with such a title`}
```